src/app/search/search.component.html

```diff
@@ -1,4 +1,4 @@
-<h3 class="heading-medium">Search</h3>
+<h3 class="heading-medium" style="margin-top: 25px;">Search</h3>
```

Comment:
It could be nice to have it as a class in scss and added it in the class eg:-class= "heading-medium search-margin" instead of margin-top: 25px;

Reply:
Done.